tests/robot-tests/tests/admin/bau/prerelease.robot

```diff
@@ -0,0 +1,99 @@
+    user waits until page contains heading 2  Release summary
+    user checks summary list contains  Publication title  ${PUBLICATION_NAME}
+
+Go to "Release status" tab
+    [Tags]  HappyPath
+    user clicks link   Release status
+    user waits until page contains heading 2  Release status
+    user waits until page contains button  Edit release status
+
+Approve release
+    [Tags]  HappyPath
+    ${PUBLISH_DATE_DAY}=         get datetime  %d  2
+    ${PUBLISH_DATE_MONTH}=       get datetime  %m  2
+    ${PUBLISH_DATE_MONTH_WORD}=  get datetime  %B  2
+    ${PUBLISH_DATE_YEAR}=        get datetime  %Y  2
+    set suite variable  ${PUBLISH_DATE_DAY}
+    set suite variable  ${PUBLISH_DATE_MONTH}
+    set suite variable  ${PUBLISH_DATE_MONTH_WORD}
+    set suite variable  ${PUBLISH_DATE_YEAR}
+
+    user clicks button  Edit release status
+    user clicks radio   Approved for publication
+    user enters text into element  id:releaseStatusForm-internalReleaseNote     Approved by prerelease UI tests
+    user waits until page contains element   xpath://label[text()="On a specific date"]/../input
+    user clicks radio   On a specific date
+    user waits until page contains element   xpath://*[text()="Publish date"]
+    user enters text into element  id:releaseStatusForm-publishScheduled-day    ${PUBLISH_DATE_DAY}
+    user enters text into element  id:releaseStatusForm-publishScheduled-month  ${PUBLISH_DATE_MONTH}
+    user enters text into element  id:releaseStatusForm-publishScheduled-year   ${PUBLISH_DATE_YEAR}
+    user enters text into element  id:releaseStatusForm-nextReleaseDate-month   1
+    user enters text into element  id:releaseStatusForm-nextReleaseDate-year    2001
+    user clicks button   Update status
+
+Wait for release process status to be Scheduled
+    [Tags]  HappyPath
+    user waits until page contains heading 2  Release status
+    user checks summary list contains  Current status  Approved
+    user checks summary list contains  Scheduled release  ${PUBLISH_DATE_DAY} ${PUBLISH_DATE_MONTH_WORD} ${PUBLISH_DATE_YEAR}
+    user checks summary list contains  Next release expected  January 2001
+    user waits for release process status to be  Scheduled  90
+    # Bug?
+    #user checks page does not contain button  Edit release status
+
+Navigate to Prerelease page
+    [Tags]  HappyPath
+    ${current_url}=  get location
+    ${release_url}=  remove substring from right of string  ${current_url}  /status
+    ${prerelease_url}=   evaluate  "${release_url}" + "/prerelease"
+    user goes to url   ${prerelease_url}
+
+Validate Prerelease page is displayed
+    [Tags]  HappyPath
+    user waits until page contains heading 1  Pre Release access is not yet available
+    user checks element count is x  css:[data-testid="breadcrumbs--list"] li     2
+    user checks element should contain  css:[data-testid="breadcrumbs--list"] li:nth-child(1)   Home
```

Comment:
Let's add some common keywords for breadcrumbs

Reply:
As there is stuff for breadcrumbs all over the UI tests, I'll create a separate ticket to do this and do it as part of that rather than as part of this PR.